packages/@aws-cdk/aws-cognito/test/integ.user-pool-explicit-props.expected.json

```diff
@@ -850,7 +848,7 @@
     "myuserpoolmyuserpooldomainEE1E11AF": {
       "Type": "AWS::Cognito::UserPoolDomain",
       "Properties": {
-        "Domain": "myawesomeapp",
+        "Domain": "myawesomeapp2",
```

Comment:
any reason you've made this change?

Reply:
It was conflicting with another test , I have reverted it back.